src/ccalltest.c

```diff
@@ -292,6 +292,8 @@ typedef struct {
     complex double r89;
 } struct_huge4b;
 
+// Intel compiler does not currently support complex int (which is GNU extension)
+#ifndef __INTEL_COMPILER
```

Comment:
we have a `_COMPILER_INTEL_` define from `src/support/platform.h` for uniformity with other compiler vendor checks